client/client.go

```diff
@@ -146,6 +146,11 @@ func NewClient(baseDir string) (*Client, error) {
 		return nil, fmt.Errorf("error reading CT file: %v", err)
 	}
 
+	// Create the initial sync file.
+	if err := c.updateSyncFile(); err != nil {
```

Comment:
The client isn't supposed to be updating a file at boot if the file already exists, it should only be updating the file at boot if the file doesn't exist yet. So this would need extra logic around it checking for the presence of the file, and that logic should probably be broken out into its own helper function